src/controllers/flamegraphController.ts

```diff
@@ -24,15 +26,20 @@ type FlamegraphSVGController = {
 const flamegraphController: FlamegraphSVGController = {
   stackCollapse: (req: Request, res: Response, next: NextFunction): void => {
     //node child process
-    const fileName = res.locals.id; 
-    console.log(fileName, '-----')
-    exec(`./src/perlScripts/stackCollapse-perf.pl ./database/captures/${fileName}.perf > ./database/folded/${fileName}.folded`,
-       (error: stream.Readable, stdout: stream.Readable, stderr: stream.Readable) => {
+    console.log('---dirname---', __dirname); // ...src/controllers/flamegraphcontroller.ts
+    const fileName: string = res.locals.id; 
+    const inputPath: string = path.resolve(__dirname, `../../database/captures/${fileName}.perf`)
+    const outputPath: string = path.resolve(__dirname, `../../database/folded/${fileName}.folded`)
+    const script: string = path.resolve(__dirname, `../../src/perlScripts/stackCollapse-perf.pl`)
```

Comment:
We'll need to change this when we actually build a distributable npm module, as it won't include `src`, only things that are in `dist`. We'll also have to add a build task to copy the `pl` script over to the right place in `dist` as part of the build.